### ISSUE TITLE
[Designer Accessibility] Bind designer peer button should be keyboard accessible

### ISSUE THREAD

**Issue by anna-dingler** [microsoft/AdaptiveCards#7794]
Keyboard accessibility needs to be enabled for the bind button on designer peers.

<img width="404" alt="image" src="https://user-images.githubusercontent.com/98650930/187734941-4c899eed-c153-49d9-a89a-fecd6e384575.png">

See #7159
